src/ucp/api/ucp.h

```diff
@@ -1457,26 +1457,32 @@ void ucp_worker_wait_mem(ucp_worker_h worker, void *address);
  *
  * @code {.c}
  * void application_initialization() {
+ * // should be called once in application init flow and before
```

Comment:
can you please put the doc update in another PR? we want to review the api/doc change separately and not make the rdma_cm wakeup fix/test depend on the doc fix